documentation/book/assembly-kafka-connect.adoc

```diff
@@ -9,15 +9,13 @@
 [id='kafka-connect-{context}']
 = Kafka Connect
 
-The Cluster Operator deploys a link:https://kafka.apache.org/documentation/#connect[Kafka Connect^] cluster, which can be used with your Kafka broker deployment.
-It is implemented as a `Deployment` with a configurable number of workers.
-The default image currently contains only the `FileStreamSinkConnector` and `FileStreamSourceConnector` connectors.
-The REST interface for managing the Kafka Connect cluster is exposed internally within the {ProductPlatformName} cluster as a `kafka-connect` service on port `8083`.
-
-Example `KafkaConnect` resources and the details about the `KafkaConnect` format for deploying Kafka Connect can be found in
-xref:assembly-deployment-configuration-kafka-connect-str[Kafka Connect cluster configuration]
-and
-xref:assembly-deployment-configuration-kafka-connect-s2i-str[Kafka Connect cluster with Source2Image support].
+link:https://kafka.apache.org/documentation/#connect[Kafka Connect^], a core component of Apache Kafka, is a tool for streaming high volumes of data into and out of your Kafka cluster. It provides a framework for building connectors (a type of plug-in) for your Kafka cluster, a REST API for managing connectors, and more. Kafka Connect includes two built-in connectors, which are both available for use in {ProductName}:
+
+- A source connector, named `FileStreamSourceConnector`.
```

Comment:
Maybe is it better saying "a file source connector" ?

Reply:
Decided to present the information in a table rather than a bullet list. The header rows are **File Connector** and **Description**.